src/types/reg/udral/physics/acoustics/Note_0_1.hpp

```diff
@@ -5,9 +5,9 @@
 // Checking this file under version control is not recommended since metadata in this header will change for each
 // build invocation. TODO: add --reproducible option to prevent any volatile metadata from being generated.
 //
-// Generator:     nunavut-2.3.1 (serialization was enabled)
```

Comment:
I'd say you are using quite an outdated nnvg version. Have you used the script supplied with this library?

Reply:
I used the update_cpp_header.sh file in the extras folder to generate

Reply:
How about you follow the instructions outlined here: https://github.com/107-systems/107-Arduino-Cyphal/blob/main/extras/script/README.md 🙏

Reply:
my apologies